internal/backends/rlang/rlang.go

```diff
@@ -0,0 +1,138 @@
+package rlang
+
+import (
+	"fmt"
+	"regexp"
+	"github.com/ALANVF/rgo"
+	
+	"github.com/replit/upm/internal/api"
+	"github.com/replit/upm/internal/util"
+)
+
+func getImports(imports string) []string {
+	return regexp.MustCompile(`[a-zA-Z_]\w*`).FindAllString(imports, -1)
+}
+
+func createRPkgDir() {
+	if _, err := os.Stat("~/R"); os.IsNotExist(err) {
+		if err := os.MkdirAll("~/R/x86_64-pc-linux-gnu-library/3.4", os.ModeDir); err != nil {
```

Comment:
We often put packages in the working directory of the running process. Intermediate stuff we can go elsewhere (for example, Maven uses `~/.m2/repository`). It really depends on the convention for the package manager!

Reply:
Ok I can change that. I had done that because that is the directory that R defaults to if it can't install anywhere else